modules/client/shared/src/main/scala/com/azavea/stac4s/api/client/BaseSttpStacClient.scala

```diff
@@ -0,0 +1,105 @@
+package com.azavea.stac4s.api.client
+
+import com.azavea.stac4s.{StacCollection, StacItem}
+
+import cats.MonadError
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import cats.syntax.option._
+import eu.timepit.refined.types.string.NonEmptyString
+import io.circe.syntax._
+import io.circe.{Encoder, Json}
+import sttp.client3.circe.asJson
+import sttp.client3.{SttpBackend, basicRequest}
+import sttp.model.Uri
+
+abstract class BaseSttpStacClient[F[_]: MonadError[*[_], Throwable]](
```

Comment:
hm, mb instead of `Base` we can name it `SttpStacClientF`?

Reply:
I'm good with that -- I think it's more descriptive than `Base`. `Base` sounds like a setup for some 🍰